curations/npm/npmjs/-/source-map.yaml

```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: source-map
+  provider: npmjs
+  type: npm
+revisions:
+  0.1.43:
+    licensed:
+      declared: BSD-3-Clause-Attribution
```

Comment:
There are (7) options for BSD-3 clause and I didn't see the plain "BSD-3 clause" at the bottom of the list.  And honestly, I don't know what all the (7) options mean.

Reply:
If I counted right, there are 17 options for “BSD” in the ClearlyDefined drop down menu.  Is there a cheat sheet for what these all mean?

Reply:
@capfei - It is not https://spdx.org/licenses/BSD-3-Clause-Attribution.  Do I need to submit a new Contribution for just "BSD-3 clause" or you will fix?

Reply:
The license list comes from SPDX. It lists the full name, the SPDX identifier (what is listed in the dropdown), and a link to the license text. Let me know if that helps. This might be something that could use some UI tweaking.  

https://spdx.org/licenses/

Reply:
@capfei - Yes, I am more familiar with the OSI approved licenses than I am with all the SPDX variations.  Thanks for the link.  So, do I need to resubmit a new Contribute request?

Reply:
You can or you can edit this one. To edit, click on "Files changed" tab at the top by "Conversations" and "Commits", or go to https://github.com/clearlydefined/curated-data/pull/172/files. You should see a pencil icon on the right side, next to the trash can. This should let you edit the file. Once you made the change, add a change message at the bottom, leave it as "Commit directly...", and commit your changes.

If you any problems, let me know and I can make the change for you.

Reply:
@capfei - cool, though it says I don't have push access.  I guess you need to make the change?

Reply:
Sorry, I thought you would be able to update your pull request. 😞 I'll get it done and merged. Thank you for the contribution!